TestFoundation/TestBundle.swift

```diff
@@ -36,13 +36,70 @@ internal func xdgTestHelperURL() -> URL {
 
 
 class BundlePlayground {
+    enum ExecutableType: CaseIterable {
+        case library
+        case executable
+        
+        var pathExtension: String {
+            switch self {
+            case .library:
+                #if os(macOS) || os(iOS) || os(watchOS) || os(tvOS)
+                return "dylib"
+                #elseif os(Windows)
+                return "dll"
+                #else
+                return "so"
+                #endif
+            case .executable:
+                #if os(Windows)
+                return "exe"
+                #else
+                return ""
+                #endif
+            }
```

Comment:
I wonder if we want to have some constants for this in `FileManager` as we keep wanting to do this.

Reply:
Yea :\ probably a SPI picking up that information from CF (like: `Bundle._platformLibraryPrefix/Suffix`, `Bundle._platformExecutablePrefix/Suffix` or similar).